vsts/determine_tests_to_run.ps1

```diff
@@ -15,6 +15,7 @@ if (($env:TARGET_BRANCH).toLower().Contains("system.pullrequest.targetbranch"))
 
     $Env:runIotHubTests = "true"
     $Env:runProvisioningTests = "true"
+	$Env:runDigitalTwinTests = "true"
```

Comment:
This and below is showing off on indentation but I checked in intellij it looks all good.

Reply:
That's because it is a tab instead of spaces. You can make them consistent and it'll look right in both IntelliJ and the web.